proc/proc.go

```diff
@@ -130,6 +135,13 @@ func GetKernelModules(modulesPath string,
 			Size:    int(size),
 		})
 	}
+
+	if !atLeastOneValidAddress {
+		return nil, fmt.Errorf(
+			"addresses from modules is zero - "+
+				"check process permissions: '%s'", lastInvalidAddressLine)
```

Comment:
Does the `lastInvalidAddressLine` give any additional information here? We could just say "all module addresses are zero" and drop the line numbers as irrelevant.

Reply:
removed